spec/requests/graphql/mutations/digital_object/create_digital_object_spec.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Create Digital Object', type: :request, solr: true do
+  let(:project) { FactoryBot.create(:project) }
+
+  let(:field_definitions) do
+    {
+      dynamic_field_categories: [{
+        display_label: "Descriptive Metadata",
+        metadata_form: 'descriptive',
+        dynamic_field_groups: [
+          {
+            string_key: 'title',
+            display_label: 'Title',
+            dynamic_fields: [
+              { display_label: 'Non-Sort Portion', sort_order: 1, string_key: 'non_sort_portion', field_type: DynamicField::Type::STRING },
+              { display_label: 'Sort Portion', sort_order: 2, string_key: 'sort_portion', field_type: DynamicField::Type::STRING }
+            ]
+          }
+        ]
+      }]
+    }
+  end
+
+  before do
+    Hyacinth::DynamicFieldsLoader.load_fields!(field_definitions)
+    search_params = { digital_object_type: 'item' }
+    EnabledDynamicField.where(search_params).delete_all
```

Comment:
It's not clear to me why the above `delete_all` line runs before each test.  Wouldn't the db be cleared before each test anyway?

Reply:
Yep.

Reply:
```suggestion
```